src/components/Badge.vue

```diff
@@ -57,72 +57,62 @@ export default {
 	&__container {
 		border-radius: 50px !important;
 		width: fit-content;
-		padding: pYX(0, 2);
+		padding: pYX(1, 3);
 	}
 
 	&__content {
-		font-weight: 500 !important;
-		@include body-2;
+		@include subheading-2;
```

Comment:
Na versão 3.0.0, a atualização de estilo utiliza `body-1`, a diferença é apenas o `letter-spacing`

```suggestion
		@include body-1;
```

Reply:
![Animação-3](https://user-images.githubusercontent.com/45737248/189979918-d8450aa7-bbaf-4bb6-9f49-f2b4e038198a.gif)

Reply:
é bom alinhar isso com Rafa porque esse espaçamento não tá explícito no mockup.